GsaGH/GsaGHInfo.cs

```diff
@@ -16,37 +16,39 @@
 using Utility = OasysGH.Utility;
 
 namespace GsaGH {
-  public static class SolverRequiredDll {
```

Comment:
static stuff should be on top.. or its own class

Reply:
or is there a different reason for that?

Reply:
should be its own class